styles/index.css

```diff
@@ -31,18 +32,15 @@ body {
     --dark-black: black;
     --white-color: white;
     --nav-floating: #ff4141;
-
     --footer-color: #111;
     --light-text: #9e9e9e;
     --transparent-layout: #c8c8c866;
-
     --footer-color: #111;
     --light-text: #9e9e9e;
     --transparent-layout: rgba(200, 200, 200, 0.4);
```

Comment:
Suggestion: You can convert this rgba value to hex (for consistency) using online tools.

Reply:
will do